torch/__init__.py

```diff
@@ -511,11 +511,23 @@ def _sympy_(self):
         return self.node.expr
 
     def __hash__(self) -> builtins.int:
+        return hash(self._get_int())
+
+    def as_integer_ratio(self) -> _Tuple[builtins.int, builtins.int]:
+        """Represent this int as an exact integer ratio"""
+        return self._get_int(), 1
+
+    def bit_length(self) -> "SymInt":
+        return SymInt(self.node.wrap_int(self._get_int().bit_length()))
```

Comment:
I'm going to submit a patch that fixes this, but there's no point in wrapping a constant integer (which is what you're guaranteed to get here) into a SymInt. Constant SymInt decay to int as early as possible.

Reply:
Ah, thanks, that's a very useful axiom to keep in mind.